test/functional/Java19andUp/src/org/openj9/test/jep425/VirtualThreadTests.java

```diff
@@ -66,12 +66,14 @@ public void test_VirtualthreadYieldResume() {
 							results[i] = 1;
 							Thread.sleep(Duration.ofSeconds(1));
 							results[i] += 1;
+							Thread.sleep(Duration.ofSeconds(1));
+							results[i] += 1;
 							return i;
 					});
 			});
 			executor.awaitTermination(2L, TimeUnit.SECONDS);
```

Comment:
Will have to increase the timeout value here (to `3L` maybe) to make sure vthread actually finish rather than killed by scheduler

Reply:
may be a loop is better; because timing will also vary on different platforms:
```
// MAX_TRIES == 5
for (int tries = 0; (tries < MAX_TRIES) || !executor.awaitTermination(2L, TimeUnit.SECONDS); i++) {
}
AssertJUnit.assertTrue("Executor tasks not completed.", executor.isTerminated())
```

Reply:
Updated.